jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3839,12 +3855,20 @@ private final static boolean StrHWAvailable() {
 		return STRING_OPT_IN_HW;
 	}
 
-	// DO NOT CHANGE CONTENTS OF THESE METHODS
-	private final boolean toUpperHWOptimized(String input) {
+	// DO NOT CHANGE THE CONTENTS OF THESE METHODS
+	private final static boolean toUpperHWOptimizedLatin1(byte[] value, byte[] output, int length) {
```

Comment:
Let's move this guy to JitHelpers as well and fully document it. Similarly with others.